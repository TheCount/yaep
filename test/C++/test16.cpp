#include <stdlib.h>

#include"common.h"

static const char *input = "a+a*(a*a+a)";

static const char *description =
"\n"
"E : T         # 0\n"
"  | E '+' T   # plus (0 0)\n"
"  ;\n"
"T : F         # 0\n"
"  | T '*' F   # mult (0 2)\n"
"  ;\n"
"F : 'a'       # 0\n"
"  | '(' E ')' # 1\n"
"  ;\n"
  ;

int
main ()
{
  test_standard_parse (input, description);
  exit (0);
}
