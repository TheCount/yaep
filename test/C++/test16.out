repeated translation symbol number 0 in rule for `E'
